Essential/001_Start_HomeWork.htm

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Any files from another tasks are redundant

Reply:
About another tasks - I read comment above and I haven't any objections about those files